.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,7 @@
 - [ ] New entries are in format `SOURCE DESTINATION TAG`
 - [ ] New entries are added to the correct section of the list (sorted lexicographically)
 - [ ] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
+- [ ] New entries have licenses of images listed here: TODO
```

Comment:
```suggestion
- [ ] New entries are licensed with Rancher [favored or approved licenses](https://github.com/rancherlabs/rancher-security/wiki/Rancher-Licensing-Standards)
```

Reply:
Addressing @superseb comment.

Reply:
Can we publish this somewhere externally-facing? This is a public repo. Hiding documentation supporting a checklist item in a private repo feels wrong.

Reply:
I purposefully left this is a "list" question, as opposed to a checkbox question, so that people actually do the hard work of listing all licenses out. A checkbox is too easy to check and move on, we have seen lots of that already.
We could also talk about making our guidelines public, but that has wider implications.

Reply:
@brandond I don't think that the place to list the licenses was already defined. Nevertheless, I'm checking regarding listing the "favored or approved licenses", as you mentioned.